### ISSUE TITLE
NS2003 false-positive when InternalsVisibleTo is defined in .csproj

### ISSUE THREAD

**Issue by rklec** [nsubstitute/NSubstitute.Analyzers#210]
## STR

* Have an internal interface in your project. 
* Have another test project, where you substitute it.

Have this in your `.csproj`
```xml
  <ItemGroup>
    <InternalsVisibleTo Include="ProjectName.Test" />
    <InternalsVisibleTo Include="ProjectName.IntegrationTest" />
  </ItemGroup>  
```

### What happens

The issue is you anyway get the error:
> [NS2003](https://github.com/nsubstitute/NSubstitute.Analyzers/blob/master/documentation/rules/NS2003.md) Can not substitute for internal type. To substitute for internal type expose your type to DynamicProxyGenAssembly2 via [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]

![grafik](https://github.com/nsubstitute/NSubstitute.Analyzers/assets/71322635/17450ca1-d85d-4701-9e22-6e0bdf81cb83)

### What should happen

No error. After all I can import and use it, this kind-of proves that this works.. Also the tests still execute.

The correction it suggests is also adding `InternalsVisibleTo`, but I already did that - it just suggests the C# code version.

## System
.NET 6
```xml
    <PackageReference Include="NSubstitute.Analyzers.CSharp" Version="1.0.16">
```

**Comment by rklec** [comment]
Ah, forget it, I should read, you need to expose it to your assembly, too:

```xml
    <InternalsVisibleTo Include="DynamicProxyGenAssembly2" /> 
```

This also removes the linting/analyzer warning. Sorry.
